nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -743,6 +766,23 @@ CREATE ROLE postgres LOGIN SUPERUSER;
      See <link xlink:href="https://github.com/NixOS/nixpkgs/pull/82743#issuecomment-674520472">the PR that changed this</link> for more info.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       The <link linked="opt-services.mediatomb">mediatomb service</link> is now
+       using by default the new fork and maintained <literal>gerbera</literal>
+       service instead of the unmaintained <literal>mediatomb</literal>
+       package. If you want to keep the old behavior, you must declare it with:
+       <programlisting>
+       services.mediatomb.package = pkgs.mediatomb;
+       </programlisting>
+       One new option <literal>openFirewall<literal> has been introduced which
+       defaults to false. If you relied on the service to open the firewall
+       rules itself, you should now declare it with:
```

Comment:
```suggestion
       itself, you should now declare it with:
```

(I think it should either be "open the firewall" or "add the firewall rules", but not a combination of both).